pkg/nsx/services/firewall.go

```diff
@@ -581,7 +1159,7 @@ func (service *SecurityPolicyService) CreateOrUpdateSecurityPolicy(obj *v1alpha1
 
 	if len(nsxSecurityPolicy.Scope) == 0 {
 		// warning
-		log.Info("SecurityPolicy %s has empty policy-level appliedTo")
```

Comment:
Thanks for the fix. I found it but forgot to fix...

Reply:
you are welcome:)